src/js/script.js

```diff
@@ -1,5 +1,3 @@
-import 'babel-polyfill';
-
```

Comment:
That may be because of the `last 2 versions` target. It matches IE 10 as well, unfortunately:
http://browserl.ist/?q=last+2+versions